From 474179c87ab0ff562fdfd2471b02797e4bdd3148 Mon Sep 17 00:00:00 2001
From: Jerome Benoit <quartermaster@rezozer.net>
Date: Sat, 4 Nov 2023 23:22:30 +0100
Subject: [PATCH] cythonize: discard now obsoleted directive

Description: upstream: fix: obsoleted compiler directive: discard
 This patch discards the now obsolete compiler directive
 `legacy_implicit_noexcept`.
Origin: vendor, Debian
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2023-11-04

This is https://github.com/sagemath/cysignals/pull/193

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index eb9c8f5a..79531705 100755
--- a/setup.py
+++ b/setup.py
@@ -173,7 +173,6 @@ def cythonize(self, extensions):
                 compiler_directives=dict(
                     binding=True,
                     language_level=2,
-                    legacy_implicit_noexcept=True,
                     ))
 
 
